From 7b4dd21630afa835a907cd030fd4e2af6849c6a9 Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@dell.com>
Date: Tue, 26 Sep 2017 13:50:02 -0500
Subject: [PATCH] platform/x86: dell-smbios: Add pr_fmt definition to driver
Git-commit: 7b4dd21630afa835a907cd030fd4e2af6849c6a9
Patch-mainline: v4.15-rc1
References: FATE#325842

pr_fmt provides formatting to be used by the driver when
displaying errors and messages.

Signed-off-by: Mario Limonciello <mario.limonciello@dell.com>
Reviewed-by: Pali Pohar <pali.rohar@gmail.com>
Signed-off-by: Darren Hart (VMware) <dvhart@infradead.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/platform/x86/dell-smbios.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/platform/x86/dell-smbios.c
+++ b/drivers/platform/x86/dell-smbios.c
@@ -12,6 +12,7 @@
  *  it under the terms of the GNU General Public License version 2 as
  *  published by the Free Software Foundation.
  */
+#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
 
 #include <linux/kernel.h>
 #include <linux/module.h>
